src/internal/pfsdb/branches_test.go

```diff
@@ -428,7 +446,7 @@ func TestBranchTrigger(t *testing.T) {
 			// Attempt to create trigger with nonexistent branch via UpsertBranch
 			gotMasterBranchInfo.Trigger = &pfs.Trigger{Branch: "nonexistent"}
 			_, err = pfsdb.UpsertBranch(ctx, tx, gotMasterBranchInfo)
-			require.True(t, errors.Is(err, pfsdb.ErrBranchNotFound{BranchKey: "project1/repo1.user@nonexistent"}))
+			require.True(t, errors.As(err, &pfsdb.ErrBranchNotFound{BranchKey: "project1/repo1.user@nonexistent"}))
```

Comment:
nit: when using `errors.As` only the type matters, so we don't really need to populate the struct fields.